packages/smart-camera-web/smart-camera-web.js

```diff
@@ -1494,25 +1496,65 @@ class SmartCameraWeb extends HTMLElement {
     }
 
     if (this.takeDocumentPhotoButton)
-      this.takeDocumentPhotoButton.addEventListener('click', () =>
-        this._startIDCamera(),
-      );
+      this.takeDocumentPhotoButton.addEventListener('click', () => {
+        this._startIDCamera();
+        console.log('dispatch: metadata:document-front-capture-start');
+        window.dispatchEvent(
+          new CustomEvent('metadata:document-front-capture-start'),
+        );
+        console.log('dispatch: metadata:document-front-origin', {
+          detail: { imageOrigin: 'camera' },
+        });
+        window.dispatchEvent(
+          new CustomEvent('metadata:document-front-origin', {
+            detail: { imageOrigin: 'camera' },
+          }),
```

Comment:
Whenever we have a capture start, we also have a corresponding event for the camera name. we can probably merge them. what do you think?